staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -575,6 +576,16 @@ func installAPI(s *GenericAPIServer, c *Config) {
 		if c.EnableContentionProfiling {
 			goruntime.SetBlockProfileRate(1)
 		}
+		// so far, only logging related endpoints are considered valid to add for these debug flags.
+		routes.DebugFlags{}.Install(s.Handler.NonGoRestfulMux, "v", routes.StringFlagPutHandler(
```

Comment:
Why is this gated on profiling being enabled?

Reply:
That's @lavalamp suggested: https://github.com/kubernetes/kubernetes/pull/63777#discussion_r191838475